test/login.e2e-spec.ts

```diff
@@ -0,0 +1,80 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as faker from 'faker';
+import { StatusCodes } from 'http-status-codes';
+import * as request from 'supertest';
+import { factory, useRefreshDatabase, useSeeding } from 'typeorm-seeding';
+import { AppModule } from '../src/app.module';
+import { User } from '../src/entities/user.entity';
+
+describe('A user make a request to the authentication in the system', () => {
+  let app: INestApplication;
+
+  // TODO: change payload type to Partial<LoginDTO>
```

Comment:
Since Login DTO is not used here, delete this `TODO`.

Reply:
Resolved on 7ff3f7ee216842cd7af78536a1ce255a2224db7a.